### DISCUSSION TITLE
Binary feature problem

### DISCUSSION THREAD

**Discussion by eduard-ignatev** [alan-turing-institute/sktime#3848] [Q&A]
Hi,

So I have a binary feature ('status_stop') that basically indicate that the target should be close to zero. But for some reason it works the other way around with sktime (screenshot №2). If I simply build the same model with tabular regression without sktime, I get the expected response (screenshot №3). Seems like I am missing something obvious here, could anyone help?

```
# Forecasting horizon is 144 intervals (10min) ahead, which equals 1 day.
fh = ForecastingHorizon(np.arange(1, 145), is_relative=True)

# Data up to '2021-03-31 23:50:00' will be used for training and validation, the rest is for the demo unit
y = df.loc[:'2021-03-31 23:50:00', ['power']]
X = df.loc[:'2021-03-31 23:50:00'].drop('power', axis=1)

# Define forecaster
cb = make_reduction(CatBoostRegressor(verbose=1000), scitype="tabular-regressor")

# Fit CatBoost regressor
cb.fit(y=y, X=X)

# Assume we're in a new point of time in future
y_observed = df.loc[:'2021-05-24 23:50:00', ['power']]
X_observed = df.loc[:'2021-05-24 23:50:00'].drop('power', axis=1)

# Update forecaster with new data
cb.update(y_observed, X_observed, update_params=False)

# True target day-ahead
y_true = df.loc['2021-05-25 00:00:00':'2021-05-25 23:50:00', ['power']]

# Features day-ahead
X_forecast = df.loc['2021-05-25 00:00:00':'2021-05-25 23:50:00'].drop('power', axis=1)

# Predicted target day-ahead
y_forecast = cb.predict(fh, X_forecast)
```
![Screenshot from 2022-11-28 23-50-51](https://user-images.githubusercontent.com/97912967/204378761-302f42a8-78db-442f-9dbc-0e29125ad9c9.png)
![Screenshot from 2022-11-28 23-52-25](https://user-images.githubusercontent.com/97912967/204378774-88f94009-bff6-41a5-bcdb-52ac2b4c0bb4.png)
![Screenshot from 2022-11-28 23-53-18](https://user-images.githubusercontent.com/97912967/204378792-5dfdce4f-95fb-4c29-ae6a-139f4f860dc5.png)

**Comment by fkiraly** [comment]
could you post a self-contained piece of code, with all imports and some dummy data? Then we can look into this to help you.

Might be a bug.

**Reply by eduard-ignatev** [reply]
Here is the notebook and dataframe for reproduction

[binary-feature-problem.zip](https://github.com/sktime/sktime/files/10164936/binary-feature-problem.zip)
